backend/tests/v5/e2e/routes/teamspaces/projects/containers/revisions.test.js

```diff
@@ -88,14 +92,15 @@ const setupData = async () => {
 		ServiceHelper.db.createUser(nobody),
 		ServiceHelper.db.createProject(teamspace, project.id, project.name, models.map(({ _id }) => _id)),
 		...revisions.map((revision) => ServiceHelper.db.createRevision(teamspace, modelWithRev._id, revision)),
+		ServiceHelper.db.createRevisionRef(teamspace, modelWithRev._id, `${validRefTypeRevision._id}P2006823-BDP-XX-A-Stanmore-Envelope_optimized_ifc`),
 	]);
 };
 
 const formatRevisions = (revs, includeVoid = false) => {
 	const formattedRevisions = revs
 		.sort((a, b) => b.timestamp - a.timestamp)
 		.flatMap((rev) => (includeVoid
-			|| !rev.void ? { ...rev, timestamp: rev.timestamp.getTime() } : []));
+			|| !rev.void ? { ...rev, timestamp: rev.timestamp.getTime(), rFile: undefined } : []));
```

Comment:
rFile is an internal data and it should never be exposed to the outside world, even as an undefined - check what the code is doing if you're seeing this escaping to the outside world?